iroha/src/sumeragi.rs

```diff
@@ -932,6 +923,25 @@ pub mod message {
                 signature: Signature::new(*public_key, &transaction_hash, private_key)?,
             })
         }
+
+        /// Checks that this `TransactionReceipt` is valid.
+        pub fn is_valid(&self, network_topology: &InitializedNetworkTopology) -> bool {
```

Comment:
will we use it somewhere else or it can be placed inside `block_should_be_created`

Reply:
It is not used in any other place, but I think it is better to leave it as it is for the readability purpose. It has a rather distinct purpose from `block_should_be_created`